### PR TITLE
Resolve images resize , quality and upsize .

### PR DESCRIPTION
**Pull request by SaadEddinHassania**
Resolve images resize , quality and upsize .

I used resize before, but it doesn't work , and the problem was in the condition . I resolve it and enable quality and upsize .

### CODE REVIEW

**Comment by emptynick** [comment]
(Possible) duplicate of https://github.com/the-control-group/voyager/pull/2429

**Comment by SaadEddinHassania** [comment]
but my edit for resizing too , this #2381 doesn't resolve resizing

**Comment by SaadEddinHassania** [comment]
But there is different in resizing condition

File: src/Http/Controllers/Controller.php [hunk 1]
```diff
@@ -469,6 +448,34 @@ private function is_animated_gif($filename)
         return $frames > 1;
     }
 
+    public function validateBread($request, $data)
```
Comment (fletch3555) [inline src/Http/Controllers/Controller.php]: Why did you move this method?  It does nothing except complicate the diff.  Please either provide a good reason or move it back.

**Review by fletch3555 — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/Controller.php (hunk 1 above)
Comment (SaadEddinHassania) [reply src/Http/Controllers/Controller.php]: I did move it back , but in an amend commit by wrong

**Comment by fletch3555** [comment]
#2381 is different, but definitely the same as #2538.  Closing this one as duplicate since the checks all pass on #2538
